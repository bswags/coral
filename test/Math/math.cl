
void main(){

	float a = 1.0;
	int b = 6;

	a = a+5;
	printf(a);
	printf("\n");

	a = a * (5+3);
	printf(a);
	printf("\n");

	a = a-b;
	printf(a);
	printf("\n");

	a = a/b;
	printf(a)
	printf("\n");

}
