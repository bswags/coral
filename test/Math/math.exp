6
48
42
7

