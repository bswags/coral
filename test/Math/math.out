Traceback (most recent call last):
  File "Math/math.clx", line 3, in <module>
    import coral_backend
ImportError: No module named coral_backend
